packages/commonwealth/client/scripts/views/pages/new_snapshot_proposal/new_proposal_form.ts

```diff
@@ -102,18 +106,18 @@ const newThread = async (
   form.start = Math.floor(form.start / 1000);
   form.end = Math.floor(form.end / 1000);
 
-  const version = await getVersion();
-
-  const msg: any = {
-    address: author.address,
-    msg: JSON.stringify({
-      version,
-      timestamp: (Date.now() / 1e3).toFixed(),
-      space: space.id,
-      type: 'proposal',
-      payload: form,
-    }),
-  };
+  // const version = await getVersion();
```

Comment:
What's with the commented out stuff?

Reply:
I think I realized that it never actually gets used? Both the version and msg variables are never referenced... Should I just delete entirely?